notebooks/11-xarray-intro.md

```diff
@@ -59,20 +60,28 @@ herstappe
 Or just rename the coordinate names itself:
 
 ```{code-cell} ipython3
-herstappe.rename({"x": "longitude", "y": "latitude"})
+herstappe.rename({"x": "easting", "y": "northing"})
 ```
 
-These __name dimensions__ can be used to extract (slice) data using these names making data selections very declarative:
+These __dimension coordinate names__ can be used to extract (slice) data making data selections very declarative:
 
 ```{code-cell} ipython3
 herstappe.sel(band='red')
 ```
 
+The data type of this `xarray.DataArray` `herstappe` is 'float32'. Xarray uses the data types provided by Numpy. More information on the data types Numpy supports is available in the [documentation](https://numpy.org/devdocs/user/basics.types.html#array-types-and-conversions-between-types).
+
+Converting to another data type is supported by `astype()` method:
+
+```{code-cell} ipython3
+herstappe.astype('float64')   # .nbytes
+```
+
 Using xarray:
 
-- Data stored as a Numpy arrays
-- Dimensions do have a name
-- The coordinates of each of the dimensions can represent geographical coordinates, categories, dates, ... instead of just an index
+- Data stored as a Numpy arrays.
+- Dimensions do have a coordinate with a name.
```

Comment:
```suggestion
- Dimensions do have a name.
```

(restoring the original text here, which I think is strictly speaking more correct, and coordinates are explained in the next bullet point)